src/webextension/locales/en-US/manage.ftl

```diff
@@ -1,22 +1,22 @@
 toolbar-item-count = { $count ->
-   [one]   1 item
-  *[other] { $count } items
+   [one]   1 Entry
+  *[other] { $count } Entries
```

Comment:
I think this should also be sentence case, since it's not really a title (though this one is less clear, since button labels are title case in Firefox).

Reply:
I was basing the decision for title case off of the photon design guidelines, here: http://design.firefox.com/photon/copy/capitalization.html